tabs-tabs-tabs/tabs.js

```diff
@@ -122,5 +122,12 @@ document.addEventListener("click", function(e) {
     });
   }
 
+  else if (e.target.id === "tabs-highlight") { // highlights current tab and next tab
```

Comment:
This doesn't actually work on anything but Chrome, so I think we should have some note about that somewhere, maybe in the readme or in the panel itself. 

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/highlight